--- mythtv-status-1.0.0/bin/mythtv-status~	2019-01-14 12:11:32.000000000 +0100
+++ mythtv-status-1.0.0/bin/mythtv-status	2019-01-25 22:34:40.000000000 +0100
@@ -1047,7 +1047,7 @@
     {
       # MythTV::Program currently has a slightly broken line with a numeric
       # comparision.
-      local($^W) = undef;
+      local($^W) = 0;
       $show = new MythTV::Program(@$row);
     }
 
@@ -1096,7 +1096,7 @@
     {
       # MythTV::Program currently has a slightly broken line with a numeric
       # comparision.
-      local($^W) = undef;
+      local($^W) = 0;
       $show = new MythTV::Program(@$row);
     }
 
