diff -up mythtv-status-0.9.0/bin/mythtv_recording_now.orig mythtv-status-0.9.0/bin/mythtv_recording_now
--- mythtv-status-0.9.0/bin/mythtv_recording_now.orig	2008-07-11 01:13:30.000000000 +0300
+++ mythtv-status-0.9.0/bin/mythtv_recording_now	2009-04-11 11:38:31.000000000 +0300
@@ -9,10 +9,10 @@
 # Return an exit code of 0 if the MythTV backend is currently recording
 # a show.
 
-# Where Debian stores the config info.
-if [ -f /etc/default/mythtv-status ]
+# Where Fedora stores the config info.
+if [ -f /etc/sysconfig/mythtv-status ]
 then
-  . /etc/default/mythtv-status
+  . /etc/sysconfig/mythtv-status
 fi
 
 /usr/bin/mythtv-status -h ${HOST:=localhost} --return-code-only --recording-now
diff -up mythtv-status-0.9.0/bin/mythtv_recording_soon.orig mythtv-status-0.9.0/bin/mythtv_recording_soon
--- mythtv-status-0.9.0/bin/mythtv_recording_soon.orig	2008-07-11 01:13:30.000000000 +0300
+++ mythtv-status-0.9.0/bin/mythtv_recording_soon	2009-04-11 11:39:10.000000000 +0300
@@ -9,10 +9,10 @@
 # Return an exit code of 0 if the MythTV backend is currently recording
 # a show or will be recording a show soon (in the next hour).
 
-# Where Debian stores the config info.
-if [ -f /etc/default/mythtv-status ]
+# Where Fedora stores the config info.
+if [ -f /etc/sysconfig/mythtv-status ]
 then
-  . /etc/default/mythtv-status
+  . /etc/sysconfig/mythtv-status
 fi
 
 if [ "x$1" != "x" ]
diff -up mythtv-status-0.9.0/bin/mythtv-update-motd.orig mythtv-status-0.9.0/bin/mythtv-update-motd
