--- mythtv-status-0.10.8/bin/mythtv-update-motd.orig	2018-12-23 15:52:58.351068982 +0000
+++ mythtv-status-0.10.8/bin/mythtv-update-motd	2018-12-24 00:29:02.533842709 +0000
@@ -3,12 +3,12 @@
 # Run mythtv-status to update the MOTD.
 
 NAME=mythtv-update-motd
-WORKFILE=/var/run/motd
-TEMPFILE=/var/run/motd.mythtv-status
+WORKFILE=/etc/motd
+TEMPFILE=/etc/motd.mythtv-status
 
-. /etc/default/mythtv-status
+. /etc/sysconfig/mythtv-status
 
-if [ x$RUN != xyes ]
+if [ x$UPDATEMOTD != xyes ]
 then
   exit
 fi
@@ -18,7 +18,16 @@ fi
 
 # Just incase someone has removed their motd file.
 [ -f $WORKFILE ] || touch $WORKFILE
-[ -f /var/run/motd.orig ] || cp $WORKFILE /var/run/motd.orig
+# Check if stub exists. If not, assume motd is OK to use as stub
+if [ ! -f /etc/motd.stub ]; then
+  if [ -f /etc/motd ]; then
+    cp -a /etc/motd /etc/motd.stub
+  else # No motd exists, create empty stub file
+    touch /etc/motd.stub
+  fi
+fi
+
+# Update motd:
 
 # If the tempfile is less than 15 minutes old, object, otherwise
 # we'll assume that something went wrong and remove it.
@@ -31,9 +40,9 @@ if [ -f $TEMPFILE ]; then
     fi
 fi
 
-[ ! -f /var/run/motd.orig ] && cp /var/run/motd /var/run/motd.orig
-
-cp /var/run/motd.orig /var/run/motd.new
+# Insert stub
+cp /etc/motd.stub /tmp/motd.new
+# Add mythtv status to end of stub
 ret=0
 mythtv-status $ARGS -h $HOST >> $TEMPFILE 2> /dev/null || ret=$?
 if [ $ret -eq 0 -o $ret -eq 1 ]; then
